frontend/src/stores/catalog.ts

```diff
@@ -18,7 +18,7 @@ interface State {
 
 function latestVersions<E extends CallableDTO>(data?: Array<E>): Array<E> {
     return chain(data ?? [])
-      .groupBy(func => func.module ?? `${func.module}.${func.name}`)
+      .groupBy(func => `${func.module}.${func.name}` ?? func.module)
```

Comment:
Actually this is the same as `${func.module}.${func.name}` since the `??` would never be triggered.

Reply:
ah yes, sure 🤦 , let me clean it up

Reply:
I ported a similar fix to https://github.com/Giskard-AI/giskard/pull/1348

https://github.com/Giskard-AI/giskard/pull/1348/commits/dec4a5ad8d9df11c1fe12f5a0936bf1ba51620c2